ONSdigital/takeon-egestion
# Feature/spp885/add logging to export lambda

# Description
Added spp-logger to dbexport lambda
https://collaborate2.ons.gov.uk/jira/browse/SPP-885 
Upgraded go to v 1.16 and added go.mod and go.sum files

# How to test?
In environment 1271

Send a message on the spp-es-takeon-egestion-spp1271-db-export-input queue

Example message:
{
	"snapshot_id": "14e0fb27-d450-44d4-8452-9f6996b00e27",
	"surveyperiods": [
	  {
		"survey": "023",
		"period": "201904"
	  },
	  {
		"survey": "023",
		"period": "201903"
	  },
	  {
		"survey": "066",
		"period": "201903"
	  },
	  {
		"survey": "067",
		"period": "201904"
	  }
	]
  }

Check cloudwatch logs.

Also please change log level to Debug and check
